web/source/osk/bannerManager.ts

```diff
@@ -1,6 +1,9 @@
 /// <reference path="banner.ts" />
 
 namespace com.keyman.osk {
+  /**
+   * This object is used to specify options by both `BannerManager.getOptions` and `BannerManager.setOptions`.  Refer to the latter for specification of each field.
```

Comment:
Can we make sure lines don't go over the normal 80 columns for code review and readability purposes?